tooling-docker/src/test/java/org/jboss/eap/qe/ts/common/docker/MalformedDockerConfigTest.java

```diff
@@ -83,7 +83,7 @@ public void testContainerReadyTimeout() throws Exception {
                 .build();
 
         thrown.expect(DockerTimeoutException.class);
-        thrown.expectMessage(containsString("Container was not ready in timeout"));
+        thrown.expectMessage(containsString("Container was not ready in"));
```

Comment:
This was already fixed - could you try to rebase, please?

Reply:
Rebased.